src/taxonomy/api/hooks/api.test.js

```diff
@@ -1,15 +1,24 @@
 import { useQuery } from '@tanstack/react-query';
-import useTaxonomyListData from './api';
+import { useTaxonomyListData, exportTaxonomy } from './api';
+
+const mockHttpClient = {
+  get: jest.fn(),
+};
 
 jest.mock('@tanstack/react-query', () => ({
   useQuery: jest.fn(),
+  useMutation: jest.fn(),
 }));
 
 jest.mock('@edx/frontend-platform/auth', () => ({
-  getAuthenticatedHttpClient: jest.fn(),
+  getAuthenticatedHttpClient: jest.fn(() => mockHttpClient),
+}));
+
+jest.mock('../../../utils', () => ({
+  downloadDataAsFile: jest.fn(),
```

Comment:
This also was removed
```suggestion
```

Reply:
Updated here 0be804c079df1e8964355b265cee6f02afb3335d